### DISCUSSION TITLE
Is there a way to useRowSelect in a controlled fashion?

### DISCUSSION THREAD

**Discussion by wbolduc** [TanStack/table#2847] [Q&A]
I need to be able to control which rows are selected from an external component. What is the best way to do this?

**Comment by vincent-czi** [comment]
Hardly an answer, but the best info I found on this was flipping through an [old running discussion on Spectrum](https://spectrum.chat/react-table/general/v7-how-get-selected-rows-outside-of-react-table~2deb9558-c484-4b97-9e1c-6be608f1f275). At least gives some ideas for how to approach.
